infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -325,21 +325,12 @@ public void loadedInitialStateResource(
     }
   }
 
-  public void errorIncompatibleInitialState(final UInt64 epoch) {
-    log.error(
-        "Cannot start with provided initial state for the epoch {}, "
-            + "checkpoint occurred on the empty slot, which is not yet supported.\n"
-            + "If you are using remote checkpoint source, "
-            + "please wait for the next epoch to finalize and retry.",
-        epoch);
-  }
-
   public void warnInitialStateIgnored() {
     log.warn("Not loading specified initial state as chain data already exists.");
   }
 
-  public void warnFailedToLoadInitialState(final String message) {
-    log.warn(message);
+  public void warnFailedToLoadInitialState(final Throwable throwable) {
+    log.warn("Failed to load initial state", throwable);
```

Comment:
we don't really want the stack trace if we can avoid it, do we jsut want the throwable.getMessage?